pkg/pocket/config/config.go

```diff
@@ -46,11 +47,13 @@ type Generator struct {
 
 // Config struct
 type Config struct {
-	Mode      string    `toml:"mode"`
-	DSN1      string    `toml:"dsn1"`
-	DSN2      string    `toml:"dsn2"`
-	Options   Options   `toml:"options"`
-	Generator Generator `toml:"generator"`
+	Mode        string    `toml:"mode"`
+	DSN1        string    `toml:"dsn1"`
+	DSN2        string    `toml:"dsn2"`
+	DSN3        string    `toml:"dsn3"`
+	Options     Options   `toml:"options"`
+	Generator   Generator `toml:"generator"`
+	ClientNodes []clusterTypes.ClientNode
```

Comment:
If not necessary, I wish don't save ClientNodes here, because we'd like pocket be an independent pkg.

Reply:
removed in https://github.com/pingcap/tipocket/pull/220/commits/c76b3c0660b4997e12950b06f19776546721359e and https://github.com/pingcap/tipocket/pull/220/commits/284463c7f1fe4ec8d706fb57bb211d4c0c4e859f.